code/modules/spells/ability_types/realized.dm

```diff
@@ -354,3 +354,84 @@
 		return FALSE
 	return TRUE
 
+/obj/effect/proc_holder/ability/petal_blizzard
+	name = "Petal Blizzard"
+	desc = "Increases damage taken but you make a healing aura."
+	action_icon_state = "petalblizzard0"
+	base_icon_state = "petalblizzard"
+	cooldown_time = 300 SECONDS
+	var/damage_amount = 5 // Amount of healing to plater per "pulse".
+	var/damage_count = 15 // How many times the healing is applied
+	var/damage_range = 10
+
+/obj/effect/proc_holder/ability/petal_blizzard/Perform(target, mob/user)
+
+	Pulse(user)
+	var/mob/living/carbon/human/H = user
+	to_chat(H, "<span class='userdanger'>You feel frailer!</span>")
+	H.apply_status_effect(/datum/status_effect/bloomdebuff)
+	for(var/i = 1 to damage_count - 1)
+		addtimer(CALLBACK(src, .proc/Pulse, user), i* 3)
+	return ..()
+
+/obj/effect/proc_holder/ability/petal_blizzard/proc/Pulse(mob/user)
+	new /obj/effect/temp_visual/cherry_aura(get_turf(user))
+	var/mob/living/carbon/human/H = user
+	H.adjustBruteLoss(-damage_amount)
```

Comment:
Check if they are working. Don't heal people that are working or this automatically becomes OP even before the balance pass.

Reply:
how do I do that(also Saw that one sin's realized armor does something similar so not sure if it could stay